server/service/mailService.js

```diff
@@ -0,0 +1,52 @@
+const nodemailer = require('nodemailer');
+const { getUser } = require('../service/userService');
+const moment = require('moment');
+
+const transporter = nodemailer.createTransport({
+    host: process.env.SMTP_HOST,
+    port: process.env.SMTP_PORT,
+    secure: false,
+    auth: {
+        user: process.env.SMTP_USER,
+        pass: process.env.SMTP_PASSWORD
+    }
+});
+
+const sendPendingLeaveMail = async (leave) => {
+    const requester = await getUser(leave.requesterId);
+    sendMail(process.env.APPROVAL_MAIL_RECEIVER, 'New leave notification', prepareHtmlBodyForPendingLeave(leave, requester));
+};
+
+const prepareHtmlBodyForPendingLeave = (leave, requester) => {
+    const APPROVAL_PAGE_URL = process.env.APPROVAL_PAGE_URL;
+    return '' +
+        '<h4>New leave was created and need to be approved.</h4>' +
+        '<b>Requester: </b> ' + requester.firstName + ' ' + requester.surname + '<br>' +
+        '<b>Days: </b>' + formatDate(leave.startDate) + ' - ' + formatDate(leave.endDate) + '<br>' +
+        '<b>Type: </b>' + leave.type + '<br>' +
+        ((APPROVAL_PAGE_URL) ? '<a href="' + APPROVAL_PAGE_URL + '">Approve</a>' : '');
```

Comment:
paci sa mi, ze to je conditional :) 
ale sprav to cez nieco stylu BASE_URL (ktora len ukazuje na "root") a zvysok vypocivaj. Mozno budeme posielat ine maily s inymi url...